examples/nextjs-app-router/app/auth/login/page.tsx

```diff
@@ -1,4 +1,4 @@
-import { Login } from "@ory/elements-react"
+import { Login } from "@ory/elements-react/theme"
```

Comment:
Should this be `theme/default`?

Reply:
Ok yeah, that makes sense. So then the CSS import change also needs to be reverted

Reply:
I never merged it. https://github.com/ory/elements/pull/241